README.md

```diff
@@ -51,10 +51,14 @@ To isolate and be able to re-produce the environment for this package, you shoul
 
 Then exclusively use `venv/bin/python`, `venv/bin/pip`, etc. (It is no longer recommended to use `venv/bin/activate`.)
 
-Dependencies for development are stored in requirements.txt; they are installed into the virtual environment as follows:
+If you have `tox` installed and would like it to create your environment and install dependencies for you run:
+
+    tox --devenv <name you'd like for env> -e dev
+
+Dependencies for development are specified as the `dev` `extras_requier` in `setup.cfg`; they are installed into the virtual environment as follows:
+
+    pip install --editable .[dev]
```

Comment:
Maybe we should put some quotes here so users' shells don't interpolate the `[…]` brackets?

```console
pip install --editable '.[dev]'
```

Reply:
+1, I'll update. Looks like I typoed `extras_require` as `extras_requier` as well so I'l get that in there as well.